README.md

```diff
@@ -1,10 +1,23 @@
 # Helm Charts Curated by OpenFGA
+
 Various Helm charts maintained by the OpenFGA team!
 
+[![Artifact HUB](https://img.shields.io/endpoint?url=https://artifacthub.io/badge/repository/openfga)](https://artifacthub.io/packages/helm/openfga/openfga)
+[![Discord Server](https://img.shields.io/discord/759188666072825867?color=7289da&logo=discord "Discord Server")](https://discord.com/channels/759188666072825867/930524706854031421)
+[![Twitter](https://img.shields.io/twitter/follow/openfga?color=%23179CF0&logo=twitter&style=flat-square "@openfga on Twitter")](https://twitter.com/openfga)
+
+[OpenFGA](https://github.com/openfga/openfga) is a high-performance and flexible authorization/permission engine built for developers and inspired by [Google Zanzibar](https://research.google/pubs/pub48190/).
+
+It is designed to make it easy for developers to model their application permissions and add and integrate fine-grained authorization into their applications.
```

Comment:
Should we be consistent with the description in the OpenFGA repo?

> A high-performance and flexible authorization/permission engine built for developers and inspired by [Google Zanzibar](https://research.google/pubs/pub48190/).
> 
> OpenFGA is designed to make it easy for developers to model their application permissions and add and integrate fine-grained authorization into their applications.

Reply:
The difference is that is the description for the OpenFGA repo, here it doesn't make sense to start with "A high-performance .." without adding context on what it is that we are taking about.

It would make it seem like the helm chart itself is what is being referenced.

Other than moving OpenFGA to the first sentence, the description is consistent.

Reply:
It's just common in the Helm chart community to copy/paste the project description from the main repo(s). It's weird to see a sentence start with "It is designed to make it easy .."

At minimum I'd say we should change the verbiage to "OpenFGA is designed to make it easy ..."